bedrock/base/templates/includes/protocol/footer/firefox-new.html

```diff
@@ -0,0 +1,100 @@
+            <li><a {{ fxa_link_fragment(entrypoint='mozilla.org-firefoxfooter', action='signin', utm_campaign='footer', utm_content='join-sign-in') }} class="js-fxa-cta-link" rel="external noopener"  data-link-type="footer" data-link-name="Sign In">{{ _('Sign In') }}</a></li>
+            <li><a href="{{ url('firefox.accounts') }}" data-link-type="footer" data-link-name="Benefits">{{ _('Benefits') }}</a></li>
+          </ul>
+        </section>
+        <section class="mzp-c-footer-section">
+          <h5 class="mzp-c-footer-heading">
+            {{ _('Developers') }}
+          </h5>
+          <ul class="mzp-c-footer-list">
+            <li><a href="{{ url('firefox.developer.index') }}" data-link-type="footer" data-link-name="Developer Edition">Developer Edition') }}</a></li>
+            <li><a href="{{ url('firefox.channel.desktop') + '#beta' }}" data-link-type="footer" data-link-name="Beta">Beta</a></li>
+            {# L10n: please do not translate the product name Beta #}
+            <li><a href="{{ url('firefox.channel.android') + '#beta' }}" data-link-type="footer" data-link-name="Beta for Android">{{ _('Beta for Android') }}</a></li>
+            <li><a href="{{ url('firefox.channel.android') + '#nightly' }}" data-link-type="footer" data-link-name="Nightly">Nightly</a></li>
+            {# L10n: please do not translate the product name Nightly #}
+            <li><a href="{{ url('firefox.channel.desktop') + '#nightly' }}" data-link-type="footer" data-link-name="Nightly for Android">{{ _('Nightly for Android') }}</a></li>
+          </ul>
+        </section>
+      </div>
+    </nav>
+    <nav class="mzp-c-footer-secondary">
+      <div class="mzp-c-footer-language">
+        {% include 'includes/protocol/lang-switcher.html' %}
+      </div>
+      <ul class="mzp-c-footer-links-social">
+        <li><a class="twitter" href="https://twitter.com/firefox" data-link-type="footer" data-link-name="Twitter (@firefox)">{{ _('Twitter') }}<span> (@firefox)</span></a></li>
+        <li><a class="instagram" href="https://www.instagram.com/firefox/" data-link-type="footer" data-link-name="Instagram (@firefox)">{{ _('Instagram') }}<span> (@firefox)</span></a></li>
+        <li><a class="youtube" href="https://www.youtube.com/user/firefoxchannel" data-link-type="footer" data-link-name="YouTube (@firefoxchannel)">{{ _('YouTube') }}<span> (@firefoxchannel)</span></a></li>
+      </ul>
+      <div class="mzp-c-footer-legal">
+        <p class="mzp-c-footer-license">
+          {% set moco_link = 'href="%s" data-link-type="footer" data-link-name="Mozilla Corporation"'|safe|format(url('mozorg.home')) %}
+          {% set mofo_link = 'href="https://foundation.mozilla.org" rel="external noopener" data-link-type="footer" data-link-name="Mozilla Foundation"'|safe %}
+          {% trans %}
+            Visit <a {{ moco_link }}>Mozilla Corporation’s</a> not-for-profit parent, the <a {{ mofo_link }}>Mozilla Foundation</a>.
```

Comment:
Maybe we should add the moco link to the other footer as well, so they'll share one string

Reply:
Alternatively, if this whole legal block is shared it could be its own separate include, but maybe that's going too far.

Reply:
I don't know how to resolve this. On one hand the Mozilla footer has lots of links to MoCo already. OTOH the symmetry is nice.

Can you change your maybe into a "we should" or a "nevermind"?

Reply:
We should add the link to the other footer so they will be the same string.